merkle-tree/indexed/src/array.rs

```diff
@@ -1,7 +1,16 @@
 use std::{cmp::Ordering, marker::PhantomData};
 
-use ark_ff::BigInteger;
+use ark_ff::{BigInteger, BigInteger256};
+use borsh::{BorshDeserialize, BorshSerialize};
 use light_hasher::{errors::HasherError, Hasher};
+use light_utils::be_bytes_to_bigint;
+
+#[derive(BorshDeserialize, BorshSerialize)]
+pub struct RawIndexingElement<const N: usize> {
```

Comment:
Is this to enable better offchain deserialization?

Reply:
It's for the program instruction. All types used there need to be Anchor/borsh-serializable. So using the `BigNumber` trait is not an option there